### ISSUE TITLE
wrong Block.Header.AppHash

### ISSUE THREAD

**Issue by AdJones** [smartbch/smartbch#60]
Can't sync my node - tried even from scratch and getting the same issue: 

panic: Failed to process committed block (4569914:EF1C38EA0BDDB760B74C7514B31593330E7B1B251EFD5E00422F3899AA4D3E57): wrong Block.Header.AppHash.  Expected 7DC33C437275A4BA3F5990138BDE16F98897CC9B86CF3E40A326B8FD13884401, got A191E3558F4715AD8265887B19861DACD5A4595E6A163FF46B79097FCF6533C9

It's running with the latest code, any ideas?

**Comment by techxmind** [comment]
I got the same error.  Try to remove the data and resync, but it's not working.

**Comment by AdJones** [comment]
Still haven't managed to resolve - I have tried rebuilding smartbchd and resyncing the data multiple times, but the error still occurs.

**Comment by wo4zhuzi** [comment]
I also have the same problem. How can I solve it  @helldealer

**Comment by zxh0** [comment]
A defect of smartbchd causes this problem. We are preparing a bug-fix version, it will be ready soon. We are very sorry for all the inconvenience.

**Comment by CKLFish** [comment]
I have the same problem too.
btw, Is there any snapshot we can use to sync faster?

**Comment by modenero** [comment]
same error

```
ERROR: error during handshake: error on replay: wrong Block.Header.AppHash.  Expected 7DC33C437275A4BA3F5990138BDE16F98897CC9B86CF3E40A326B8FD13884401, got A191E3558F4715AD8265887B19861DACD5A4595E6A163FF46B79097FCF6533C9
```

> @CKLFish snapshot we can use to sync faster

+1

**Comment by CKLFish** [comment]
Hi @modenero 
I tried with v0.4.4-p1 and it solved the issue~

**Comment by modenero** [comment]
> I tried with v0.4.4-p1 and it solved the issue~

fantastic!!

started re-syncing...

**Comment by zxh0** [comment]
> Hi @modenero I tried with v0.4.4-p1 and it solved the issue~

Thank you @CKLFish  
We released a bug fix version v0.4.4-p1, which can fix the issue we met last Friday.

**Comment by modenero** [comment]
> which can fix the issue we met last Friday

@zxh0 yes it does! 

my server just fully synced. 
plus i rebooted to make sure it was all good.

great work by you and your team 👏 👍
